data/js/real-time-feeds.md

```diff
@@ -6,28 +6,40 @@
 
 ## Watching Data Changes
 
-We presented a lot of features for data filtering and search. You may be wondering where the real-time aspect is in all of this. Well, it's throughout the features we just presented to you. To access our data in real-time, all we need to do is change the *WeDeploy™* API method we use to the `watch` method:
+We presented a lot of features for data filtering and search. You may be wondering where the real-time aspect is in all of this. Well, it's throughout the features we just presented to you. To access our data in real-time, all we need to do is change the *WeDeploy™* API  `get` method to use to the `watch` method:
 
 ```js
 var data = WeDeploy.data('http://datademo.wedeploy.io');
 
-data.where(Filter.any('category', 'cinema'))
-.where(Filter.distance('location', '51.5031653,-0.1123051', '1mi'))
+data.where('category', 'cinema')
```

Comment:
I would not even use a `where` clause in the basic example. We can have an advanced example that you can do that. wdyt?